packages/block-library/src/post-terms/edit.js

```diff
@@ -47,19 +42,22 @@ export default function PostTermsEdit( {
 	const { term, textAlign, separator, prefix, suffix } = attributes;
 	const { postId, postType } = context;
 
-	const selectedTerm = useSelect(
+	const post = useSelect(
+		( select ) =>
+			select( coreStore ).getEntityRecord( 'postType', postType, postId ),
+		[ postType, postId ]
+	);
+	const postTerms = post?._embedded?.[ 'wp:term' ]?.[ 0 ]?.filter(
```

Comment:
I don't like this personally, I don't think we should be relying on the "embedded" values in our code base. I would prefer if the "core-data" is smart enough to actually fill the embedded terms into its store, so the next time we fetch these terms using their dedicated selectors (usePostTerms) the request is not actually made. Would that be possible?

Reply:
I can try 😅

Reply:
Actually, I'm not sure if this PR is good at all, because the posts endpoint only supports categories and tags out of the box (hardcoded fields). So if you have custom taxonomies, it wouldn't embed anything. Doing these requests does work with custom taxonomies I think.